src/core/hle/D3D8/XbPixelShader.cpp

```diff
@@ -1284,7 +1298,9 @@ void DxbxUpdateActivePixelShader() // NOPATCH
   // as these could have been updated via SetRenderState or otherwise :
   D3DXCOLOR fColor[PSH_XBOX_CONSTANT_MAX];
   extern XboxTextureStateConverter NV2ATextureStates;
-  
+  float fogEnable, fogTableMode, fogDensity, fogStart, fogEnd;
+  float Fog_Enabled = 1;
```

Comment:
These "enabled" and "disabled" variables can be marked 'const' and could apply to many other "float booleans", so would ideally be moved to a more central location and used everywhere appropriate

Reply:
If you are going to rename them, consider `FloatTrue` vs `FloatFalse` or something along those lines.